eoscompanion/routes.py

```diff
@@ -647,11 +641,8 @@ def on_splice_finished(src, result):
                     src.splice_finish(result)
                 except GLib.Error as splice_error:
                     # Can't really do much here except log server side
-                    log(
-                        'Splice operation on file failed: {error}'.format(
-                            error=splice_error
-                        ),
-                        file=sys.stderr
+                    logging.warning(
```

Comment:
Why not `logging.error()`?

Reply:
Not sure if this is an error in the sense of the program state being inconsistent - this usually happens because the client hung up.